model/mqo/mqo2gpo.py

```diff
@@ -20,34 +36,43 @@ def __str__(self):
         return s
 
 
-class Material(object):
```

Comment:
This was identical to the Material class in the other file except for the `__str__` implementation. So, I just moved that over and got rid of the duplication.